napari/components/layerlist.py

```diff
@@ -221,16 +236,16 @@ def _get_min_and_max(self, mins_list, maxes_list):
     def _get_extent_world(self, layer_extent_list):
         """Extent of layers in world coordinates.
 
-        Default to 2D with (-0.5, 511.5) min/ max values if no data is present.
-        Corresponds to pixels centered at [0, ..., 511].
+        Default to 2D with (0, 511) min/ max values if no data is present.
+        Corresponds to image with dims 512.
```

Comment:
I'm not sure what you mean by `dims 512`. Consider the following?

```suggestion
        Corresponds to image with 512 pixels in each dimension.
```